messaging.rst

```diff
@@ -162,6 +43,8 @@ Fields
 Messages
 ========
 
+.. _direct-transfer-message:
+
 Direct Transfer
```

Comment:
@LefterisJP ,
Are we keeping the direct transfer in the specs? From what I know, we do not have it anymore.

Reply:
Update: we still have it and it works. I was not aware.